src/utils.jl

```diff
@@ -53,6 +53,10 @@ function ITensors.op(::OpName"Dup", ::SiteType"Digit", s::Index)
   return ITensor(o, s, s')
 end
 
+function ITensors.op(::OpName"Int", ::SiteType"Digit", s::Index)
```

Comment:
I think move this definition directly to the example file because it is fairly specialised / not obvious what the operator is for.